DEVELOPMENT.md

```diff
@@ -87,6 +87,16 @@ Maintain the same quality for production and test code.
 
 When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
+### Git merge strategy
+
+We prefer to `rebase and merge`, so cleaning up and squashing commits on the feature branch is best.
+`squash and merge` is reserved for when git is unfamiliar to the external contributor. 
```

Comment:
`for when` does not sound like a proper english. I think we could skip the part of `squash and merge`, it is exceptional case, we should not learn people how to leverage this path.

Reply:
Squash and merge is enabled but it does mess up the release notes checklist for some reason (because the comitter is not the person who merged the PR?). It's there for "exceptional" cases.

IMO best to remove mention of `squash and merge` as Grzegorz said.